resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 default_cow: 'base-bionic-amd64.cow'
-cows: 'base-bionic-amd64.cow'
+cows: 'base-bionic-amd64.cow base-focal-i386.cow'
```

Comment:
huh, I wonder why bionic uses `amd64` here, but all the FOSS cows use `i386`..

Reply:
Oh, good catch, I think this should be amd64. I imagine it's because we wanted to preserve 32-bit support in FOSS but not in PE. But that's just speculation 🙂